docs/deployment/image-backup.rst

```diff
@@ -116,4 +151,4 @@ way to create application-consistent backups.
 
 Borg doesn't intend to address these issues due to their huge complexity and
 platform/software dependency. Combining Borg with the mechanisms provided by the platform
-(snapshots, hypervisor features) will be the best approach to start tackling them.
\ No newline at end of file
+(snapshots, hypervisor features) will be the best approach to start tackling them.
```

Comment:
I don't see a difference here, what did you change here?

Reply:
Might be some shenanigans like a white space or so. I have no clue.